resources/qml/application.qml

```diff
@@ -193,17 +250,37 @@ ApplicationWindow {
         }
     }
 
-    Action {
-        id: quitAction
-        text: "&Quit"
-        shortcut: "Ctrl+Q"
-        onTriggered: {
-            saveAction.trigger()
-            Qt.quit()
+    function save() {
```

Comment:
What's the deal with functions vs. actions? (I just don't know. I believe you if functions are better)